pages/agent/upgrading_to_v3.md.erb

```diff
@@ -0,0 +1,94 @@
+# Upgrading to Buildkite Agent v3
+
+The Buildkite Agent has changed a lot in v3, but we've tried to keep it as backwards compatible as possible. We'll cover what’s changed and how to upgrade to new agents.
+
+<%= toc %>
+
+## What's changed
+
+* [Pipeline Plugins](/docs/agents/plugins) for easily sharing functionality between pipelines
+* The bootstrap (run as a sub-process for every job) has moved from a [shell script](https://github.com/buildkite/agent/blob/2-6-stable/templates/bootstrap.sh) to a sub-command of the `buildkite-agent bootstrap`. This means it's written in golang and is much easier for us to maintain and distribute.
```

Comment:
Also means most features work across both linux and windows.

Reply:
👍